src/ParseSessionStorage.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+namespace GrahamCampbell\Parse;
+
+use Parse\ParseStorageInterface;
+
+class ParseSessionStorage implements ParseStorageInterface
+{
+    /**
+     * Parse will store its values in a specific key.
+     *
+     * @var string
+     */
+    private $storageKey = 'parseData';
+
+    public function __construct()
+    {
+
+    }
+
+    /**
+     * Sets a key-value pair in storage.
+     *
+     * @param string $key   The key to set
+     * @param mixed  $value The value to set
+     *
+     * @return null
+     */
+    public function set($key, $value)
+    {
+        \Session::put($this->storageKey . '.' . $key, $value);
```

Comment:
We can't assume that class exists. Instead, please dependency inject the underlying instance.